WooCommerce/Classes/ViewRelated/Dashboard/Free Trial/FreeTrialSurvey/FreeTrialSurveyView.swift

```diff
@@ -0,0 +1,133 @@
+import SwiftUI
+
+final class FreeTrialSurveyHostingController: UIHostingController<FreeTrialSurveyView> {
+    init(viewModel: FreeTrialSurveyViewModel) {
+        super.init(rootView: FreeTrialSurveyView(viewModel: viewModel))
+        rootView.dismissAction = dismiss
+    }
+
+    @available(*, unavailable)
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureTransparentNavigationBar()
+    }
+
+    func dismiss() {
+        dismiss(animated: true)
+    }
+}
+
+/// View that presents a list of answers for Free trial survey
+///
+struct FreeTrialSurveyView: View {
+    @ObservedObject private var viewModel: FreeTrialSurveyViewModel
+    @FocusState private var isOtherReasonsFocused: Bool
+
+    var dismissAction: () -> Void = {}
+
+    init(viewModel: FreeTrialSurveyViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 40) {
+                Text(Localization.title)
+                    .fontWeight(.bold)
+                    .titleStyle()
+
+                VStack() {
+                    ForEach(viewModel.answers, id: \.self) { answer in
+                        if answer == .otherReasons {
+                            TextField(answer.text, text: $viewModel.otherReasonSpecified, onEditingChanged: { focus in
+                                if focus {
+                                    viewModel.selectAnswer(.otherReasons)
+                                }
+                            })
+                            .font(.body)
+                            .textFieldStyle(RoundedBorderTextFieldStyle(focused: isOtherReasonsFocused))
+                            .focused($isOtherReasonsFocused)
+                        } else {
+                            Button(action: {
+                                isOtherReasonsFocused = false
+                                viewModel.selectAnswer(answer)
+                            }, label: {
+                                HStack {
+                                    Text(answer.text)
+                                    Spacer()
+                                }
+                            })
+                            .buttonStyle(SelectableSecondaryButtonStyle(isSelected: viewModel.selectedAnswer == answer))
```

Comment:
Nit: It looks to me that the survey options have much bolder text compared to the text field for the other option. Please consider setting `.text` for `labelFont` of `SelectableSecondaryButtonStyle`.

<img width="247" alt="image" src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/9d72a8b6-b2ac-440a-98bc-e16babc040a9">

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10319/commits/0b0c5b291b061e13706aef5c6c5d19f0bf861135

<img width="237.5" alt="Screenshot 2023-07-26 at 10 48 54 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/524475/8897b07d-7bdd-46d5-b7c5-79719e84274e">